web/src/user_group_edit.js

```diff
@@ -501,8 +501,8 @@ export function show_group_settings(group) {
     setup_group_settings(group);
 }
 
-export function open_group_edit_panel_for_row(group_row) {
-    const group = get_user_group_for_target(group_row);
+export function open_group_edit_panel_for_row($group_row) {
+    const group = get_user_group_for_target($group_row);
```

Comment:
`stream_edit.ts`, which this file is intentionally very parallel to, doesn't switch everything to JQuery; does it make sense to follow the approach taken there?

Reply:
Yeah. This change is rather unnecessary. Going to fix this.